sf-docker/special/sf-with-keycloak/sf-server/sf_dependencies/docker-compose.yml

```diff
@@ -65,30 +64,29 @@ services:
       - miniodata:/data
       #- /var/lib/smartface/miniodata:/data
 
-      # Jaeger tracikng is solely for debug purposes. No data is stored!
+  # Jaeger tracikng is solely for debug purposes. No data is stored!
   jaeger:
-    image: "jaegertracing/all-in-one:1.35.0"
-    container_name: jaeger
-    restart: unless-stopped
-    command: [ "--memory.max-traces=5000" ]
-    ports:
-      - "5778:5778"
-      - "16686:16686"
-      - "14268:14268"
-      - "14250:14250"
-      - "9411:9411"
-      - "5775:5775/udp"
-      - "6831:6831/udp"
-      - "6832:6832/udp"
+   image: "jaegertracing/all-in-one:1.35.0"
+   container_name: jaeger
+   restart: unless-stopped
+   command: ["--memory.max-traces=5000"]
+   ports:
+     - "5778:5778"
+     - "16686:16686"
+     - "14268:14268"
+     - "14250:14250"
+     - "9411:9411"
+     - "5775:5775/udp"
+     - "6831:6831/udp"
+     - "6832:6832/udp"
```

Comment:
Although it is valid YAML, I think it would be better to use consistent indentation throughout each file.

Reply:
fixed I believe